nototools/create_image.py

```diff
@@ -20,9 +20,10 @@
 __author__ = 'roozbeh@google.com (Roozbeh Pournader)'
 
 import argparse
+import codecs
```

Comment:
I removed the import from the other location.  My java bias leads me to prefer putting the imports at the top of the file, even it if means we import some things we don't use unless we make a particular call.